Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -114,13 +118,15 @@ def create_context(indicators, include_dbot_score=False):
         else:
             dbot_score = 1
 
-        if include_dbot_score:
-            context['DBotScore'].append({
+        # if there is more than one indicator results - take the one with the highest score
+        if include_dbot_score and dbot_score > highest_score:
```

Comment:
indicators list is not always the same indicator by different owners, it may be different indicators